seacatauth/authz/role/service.py

```diff
@@ -316,7 +318,7 @@ async def set_roles(self, credentials_id: str, tenant_scope: set, roles: list):
 			tenant = await self.get_role_tenant(role)
 			if tenant != "*":
 				upsertor.set("t", tenant)
-			await upsertor.execute()
+			await upsertor.execute(custom_data={EventTypes.EVENT_TYPE: EventTypes.ROLES_ASSIGNED})
```

Comment:
technically, this execute call assigns a single role (there's a forloop) so it should be considered the same event type as  `ROLE_ASSIGNED` in the `_do_assign_role` method below.

please add a `TODO: Make use of the _do_assign_role method here`